### PR TITLE
Bar for all masters

### PR DESCRIPTION
**Pull request by elesbom**

### CODE REVIEW

File: dbaas/drivers/mongodb.py [hunk 1]
```diff
@@ -213,6 +213,30 @@ def list_databases(self, instance=None):
...
 
+    def update_infra_instances_sizes(self):
+        result = {
+            'updated': [],
+            'error': []
+        }
+
+        for instance in self.get_database_instances():
+            with self.pymongo(instance=instance) as client:
+                if instance.status == Instance.ALIVE:
```
Comment (otherpirate) [inline dbaas/drivers/mongodb.py]: You should do this check before create a pymongo connection

File: dbaas/drivers/mysqldb.py [hunk 2]
```diff
@@ -150,7 +182,7 @@ def info(self):
         r = self.__query("SELECT VERSION()")
         databaseinfra_status.version = r[0]['VERSION()']
 
-        my_all_dbs = self.__query("SHOW DATABASES")
+        self.__query("SHOW DATABASES")
```
Comment (otherpirate) [inline dbaas/drivers/mysqldb.py]: Is it necessary?

File: dbaas/drivers/redis.py [hunk 3]
```diff
@@ -170,6 +170,35 @@ def list_databases(self, instance=None):
...
+    def update_infra_instances_sizes(self):
+
+        result = {
+            'updated': [],
+            'error': []
+        }
+
+        for instance in self.get_database_instances():
+            with self.redis(instance=instance) as client:
+                if instance.status == Instance.ALIVE:
```
Comment (otherpirate) [inline dbaas/drivers/redis.py]: Same as mongodb driver

File: dbaas/drivers/mysqldb.py (hunk 2 above)
Comment (elesbom) [reply dbaas/drivers/mysqldb.py]: I'll ask Ricardo about that.

**Pull request merged**